flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/CodeGeneratorContext.scala

```diff
@@ -518,17 +518,6 @@ class CodeGeneratorContext(val tableConfig: TableConfig) {
     DEFAULT_TIMEZONE_TERM
   }
 
-  /**
-    * Adds a reusable Time ZoneId to the member area of the generated class.
-    */
-  def addReusableTimeZoneID(): String = {
```

Comment:
I think we can keep it... in future, we should use zoneID instead of timeZone.

Reply:
I would like to remove dead code and introduce them when we need them. Furthermore, these code does't work, for example, it uses the same member field name with timeZone, it doesn't add code to member are.

Reply:
OK